infrastructure/main.tf

```diff
@@ -7,11 +7,10 @@ module "send-letter-service-api" {
   ilbIp    = "${var.ilbIp}"
 
   app_settings = {
-    REDIS_HOST                   = "${module.redis-cache.host_name}"
-    REDIS_PORT                   = "${module.redis-cache.redis_port}"
-    REDIS_PASSWORD               = "${module.redis-cache.access_key}"
+    REDIS_HOST = "${module.redis-cache.host_name}"
+    REDIS_PORT = "${module.redis-cache.redis_port}"
+    REDIS_PASSWORD = "${module.redis-cache.access_key}"
```

Comment:
don't want to be that guy, but all other sections have values aligned vertically ;)

Reply:
No worries.Hopefully with this commit all is good :)